src/nft/components/collection/Card.tsx

```diff
@@ -166,6 +164,53 @@ const StyledImageContainer = styled.div<{ isDisabled?: boolean }>`
   cursor: ${({ isDisabled }) => (isDisabled ? 'default' : 'pointer')};
 `
 
+const StyledHoverContainer = styled.div`
+  position: fixed;
+  width: 100%;
+  /* height: 100%; */
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  padding-bottom: 1rem;
+  color: ${({ theme }) => theme.accentTextLightPrimary};
+  z-index: 1;
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.6) 0%, rgba(0, 0, 0, 0) 60%);
+  font-size: 14px;
+  font-weight: 600px;
+  aspect-ratio: 1;
+`
+const StyledToolTipContainer = styled(StyledHoverContainer)``
+
+const StyledAddAffordance = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 600px;
```

Comment:
I think this and the above is fine - I am not a huge fan of using the Link ThemedText because this is not a link and if we were to change links in the future this would get affected.

Reply:
Makes sense. Do you recall what's the name of that text component in a Figma file? AFAIK all paragraphs in Figma are referenced by the same names we use in `ThemedText` to make it easier to work on styling. If there's nothing that works, we will need to sync that file once again